rsocket/statemachine/RSocketStateMachine.h

```diff
@@ -324,7 +324,7 @@ class RSocketStateMachine final
   /// Map of all individual stream state machines.
   std::unordered_map<StreamId, std::shared_ptr<StreamStateMachineBase>>
       streams_;
-  StreamId nextStreamId_;
+  std::atomic<StreamId> nextStreamId_;
```

Comment:
Guess that should not be atomic since all the events are serialized over event bus